public/src/css/common.css

```diff
@@ -719,6 +719,16 @@ a:hover {
   z-index: 9999;
 }
 
+.pkdisplay { display: none;}
+   @media only screen
+   and (min-device-width : 320px)
```

Comment:
`min-width` is better, because `min-device-width` will not work when browser window is resized to small size on desktop.